.ci/build_demo.sh

```diff
@@ -18,6 +18,7 @@ DEMO_DOMAIN=$1
 JOB_NAME=${JOB_NAME:-$DEMO_DOMAIN}
 WORKSPACE=${WORKSPACE:-$PWD}
 ELASTICSEARCH_NETWORK=${ELASTICSEARCH_NETWORK:-shared-demo-elasticsearch-network}
+ELASTICSEARCH_INDEX_PREFIX=${JOB_NAME}
```

Comment:
```suggestion
ELASTIC_SEARCH_INDEX_PREFIX=${JOB_NAME}-
```
old implementation was also with dash
also in Dockerfile and docker-compose-demo it is ELASTIC_SEARCH_INDEX_PREFIX